packages/shared/components/pills/Pill.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+    import { Text, FontWeight, TextType } from 'shared/components'
+
+    export let data: string | undefined = undefined
+    export let textColor: string = 'gray-800'
+    export let darkTextColor: string | undefined = undefined
+    export let shrink: boolean = false
+    export let uppercase: boolean = false
+    export let invisible: boolean = false
+    export let backgroundColor: string | undefined = undefined
+    export let rounded: string = 'rounded-2xl'
+    export let padding: string = 'px-2.5 py-1'
```

Comment:
Since this is the default and used 90% of the time, I suggest removing this and adding a new variable like `size` and add variations with it

Reply:
as we agreed in slack, I'm removing `padding` prop :pray: